content/nginx-one/k8s/overview.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
I'm not convinced this needs to be its own page. Would it make sense to add this type of context to content/nginx-one/k8s/_index.md instead? Should we link to the appropriate location in the docs for each of the items in this list?